src/redux/ui/screens/input-data/validation/min-amount-to-stake.ts

```diff
@@ -0,0 +1,63 @@
+import BigNumber from 'bignumber.js';
+import { Dispatch } from 'react';
+import { IScreenFieldValidation } from '../../../../../components/widgets/types';
+import { getBlockchain } from '../../../../../core/blockchain/blockchain-factory';
+import { getChainId } from '../../../../preferences/selectors';
+import { IReduxState } from '../../../../state';
+import { getTokenConfig } from '../../../../tokens/static-selectors';
+import { IAction } from '../../../../types';
+import { getSelectedAccount } from '../../../../wallets/selectors';
+import { setScreenInputValidation } from '../actions';
+
+/**
+ * This data is stored on screen key
+ */
+export const minAmountToStake = (
+    validation: IScreenFieldValidation,
+    field: string,
+    screenKey: string,
+    getState: () => IReduxState,
+    dispatch: Dispatch<IAction<any>>
+) => {
+    const state = getState();
+
+    const account = getSelectedAccount(state);
+
+    const inputData: any = state.ui.screens.inputData[screenKey]?.data;
+
+    const blockchain = account.blockchain;
+    const blockchainInstance = getBlockchain(blockchain);
+    const chainId = getChainId(state, blockchain);
+    const token = account.tokens[chainId][blockchainInstance.config.coin];
+
+    const tokenConfig = getTokenConfig(blockchain, token.symbol);
+
+    const inputAmount = inputData?.amount;
+    const inputAmountToStd = blockchainInstance.account.amountToStd(
+        new BigNumber(inputAmount),
+        tokenConfig.decimals
+    );
+
+    // TODO
```

Comment:
we could send this value from api, inside a details field, so things would get easier.